tests/test_scope.js

```diff
@@ -1,18 +1,14 @@
 'use strict'
 
 const path = require('path')
-const { test } = require('tap')
 const { expect } = require('chai')
 const sinon = require('sinon')
 const proxyquire = require('proxyquire').noPreserveCache()
 const Interceptor = require('../lib/interceptor')
 const nock = require('..')
 const got = require('./got_client')
 
-require('./cleanup_after_each')()
-require('./setup')
```

Comment:
Don't we need the calls to setup in each file while Tap is still the runner?

Reply:
Ah, probably! I was using Mocha :)